controllers/antreainstall_controller.go

```diff
@@ -295,12 +268,11 @@ func (r *AntreaInstallReconciler) updateStatusManagerAndSharedInfo(objs []*uns.U
 		}
 		err := fmt.Errorf("configuration of resources %v is missing", missedResources)
 		log.Error(nil, err.Error())
-		operstatus.SetDegraded(r.Client, r.Status, statusmanager.OperatorConfig, "ApplyObjectsError", err.Error())
+		r.Status.SetDegraded(statusmanager.OperatorConfig, "ApplyObjectsError", err.Error())
 		return err
 	}
 	r.Status.SetDaemonSets(daemonSets)
 	r.Status.SetDeployments(deployments)
-	r.Status.SetRelatedObjects(relatedObjects)
```

Comment:
Do you think it's safe to directly remove SetRelatedObjects?

Reply:
It seems relatedobjects have been removed. I guess there's a good reason for that, but it might be worth specifying it in the commit message for future reference

Reply:
Thanks for your kind remind, and i'll check this.

Reply:
I guess it should be fine to remove `relatedObjects` as we have `sharedinfo` in operator

Reply:
It seems that relateObjects are needed for openshift, and i'll make it up.